packages/@expressive-code/plugin-collapsible-sections/src/styles.ts

```diff
@@ -0,0 +1,78 @@
+import { StyleSettings, ExpressiveCodeTheme, ResolvedCoreStyles } from '@expressive-code/core'
+
+export const collapsibleSectionClass = 'ec-section'
+
+export const collapsibleSectionsStyleSettings = new StyleSettings({
+	closedBorderWidth: '0px',
+	closedPadding: '4px 0 4px var(--padding-inline)',
+	closedMargin: '0rem',
+	closedTextColor: '',
+	closedBackgroundColor: 'rgb(84 174 255 / 20%)',
+	closedBorderColor: 'rgb(84 174 255 / 50%)',
+	openBorderWidth: '1px',
+	openPadding: '0rem',
+	openMargin: '0rem',
+	openBackgroundColor: 'rgb(246 248 250 / 7.5%)',
+	openBorderColor: 'rgb(216 222 228 / 10%)',
+})
+
+export function getCollapsibleSectionsBaseStyles(
+	theme: ExpressiveCodeTheme,
+	coreStyles: ResolvedCoreStyles,
+	styleOverrides: Partial<typeof collapsibleSectionsStyleSettings.defaultSettings>
+) {
+	const styles = collapsibleSectionsStyleSettings.resolve({
+		theme,
+		coreStyles,
+		styleOverrides,
+	})
+	const result = `
+		.${collapsibleSectionClass} {
+			& summary {
+				/* hide the default <details> marker */
+				&::marker {
+					display: inline-block;
+					content: "";
+					width: 16px;
+					height: 16px;
+				}
+
+				svg {
+					vertical-align: text-bottom;
+					fill: currentColor;
+					margin-right: 1em;
+					opacity: 0.75;
+				}
+
```

Comment:
```suggestion

				font-family: ${styles.closedFontFamily};
				font-size: ${styles.closedFontSize};
				line-height: ${styles.closedLineHeight};
				user-select: none;
				-webkit-user-select: none;

```

Add the styles that allow changing the font here, and ensure that the "x lines collapsed" texts don't get selected by accident.